pkg/driver/bigquery/bigquery_test.go

```diff
@@ -0,0 +1,252 @@
+package bigquery
+
+import (
+	"database/sql"
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/amacneil/dbmate/v2/pkg/dbmate"
+	"github.com/amacneil/dbmate/v2/pkg/dbutil"
+)
+
+func getURL() string {
+	return os.Getenv("BIGQUERY_TEST_URL")
+}
+
+func testBigQueryDriver(t *testing.T) *Driver {
+	u := dbutil.MustParseURL(getURL())
+	drv, err := dbmate.New(u).Driver()
+	require.NoError(t, err)
+
+	return drv.(*Driver)
+}
+
+func prepTestBigQueryDB(t *testing.T) *sql.DB {
+	drv := testBigQueryDriver(t)
+
+	// drop any existing database
+	err := drv.DropDatabase()
+	require.NoError(t, err)
+
+	// create database
+	err = drv.CreateDatabase()
+	require.NoError(t, err)
+
+	// connect database
+	db, err := drv.Open()
+	require.NoError(t, err)
+
+	return db
+}
+
+func TestGetDriver(t *testing.T) {
+	db := dbmate.New(dbutil.MustParseURL("bigquery://test/db_mate"))
```

Comment:
For correctness, the URL should be `bigquery:///test/db_mate`.  Although, I suspect most people will make this same mistake, so we should also support URLs that are mis-constructed, but in our code and any documentation or examples, we should make sure to use the proper form.